aws/data_aws_crossaccount_policy.go

```diff
@@ -3,149 +3,295 @@ package aws
-							"ec2:AssociateRouteTable",
-							"ec2:AttachInternetGateway",
-							"ec2:AttachVolume",
-							"ec2:AuthorizeSecurityGroupEgress",
-							"ec2:AuthorizeSecurityGroupIngress",
-							"ec2:CancelSpotInstanceRequests",
-							"ec2:CreateDhcpOptions",
-							"ec2:CreateFleet",
-							"ec2:CreateInternetGateway",
-							"ec2:CreateKeyPair",
-							"ec2:CreateLaunchTemplate",
-							"ec2:CreateLaunchTemplateVersion",
-							"ec2:CreateNatGateway",
-							"ec2:CreatePlacementGroup",
-							"ec2:CreateRoute",
-							"ec2:CreateRouteTable",
-							"ec2:CreateSecurityGroup",
-							"ec2:CreateSubnet",
-							"ec2:CreateTags",
-							"ec2:CreateVolume",
-							"ec2:CreateVpc",
-							"ec2:CreateVpcEndpoint",
-							"ec2:DeleteDhcpOptions",
-							"ec2:DeleteFleets",
-							"ec2:DeleteInternetGateway",
-							"ec2:DeleteKeyPair",
-							"ec2:DeleteLaunchTemplate",
-							"ec2:DeleteLaunchTemplateVersions",
-							"ec2:DeleteNatGateway",
-							"ec2:DeletePlacementGroup",
-							"ec2:DeleteRoute",
-							"ec2:DeleteRouteTable",
-							"ec2:DeleteSecurityGroup",
-							"ec2:DeleteSubnet",
-							"ec2:DeleteTags",
-							"ec2:DeleteVolume",
-							"ec2:DeleteVpc",
-							"ec2:DeleteVpcEndpoints",
-							"ec2:DescribeAvailabilityZones",
-							"ec2:DescribeFleetHistory",
-							"ec2:DescribeFleetInstances",
-							"ec2:DescribeFleets",
-							"ec2:DescribeIamInstanceProfileAssociations",
-							"ec2:DescribeInstanceStatus",
-							"ec2:DescribeInstances",
-							"ec2:DescribeInternetGateways",
-							"ec2:DescribeLaunchTemplates",
-							"ec2:DescribeLaunchTemplateVersions",
-							"ec2:DescribeNatGateways",
-							"ec2:DescribeNetworkAcls",
-							"ec2:DescribePlacementGroups",
-							"ec2:DescribePrefixLists",
-							"ec2:DescribeReservedInstancesOfferings",
-							"ec2:DescribeRouteTables",
-							"ec2:DescribeSecurityGroups",
-							"ec2:DescribeSpotInstanceRequests",
-							"ec2:DescribeSpotPriceHistory",
-							"ec2:DescribeSubnets",
-							"ec2:DescribeVolumes",
-							"ec2:DescribeVpcAttribute",
-							"ec2:DescribeVpcs",
-							"ec2:DetachInternetGateway",
-							"ec2:DetachVolume",
-							"ec2:DisassociateIamInstanceProfile",
-							"ec2:DisassociateRouteTable",
-							"ec2:GetLaunchTemplateData",
-							"ec2:GetSpotPlacementScores",
-							"ec2:ModifyFleet",
-							"ec2:ModifyLaunchTemplate",
-							"ec2:ModifyVpcAttribute",
-							"ec2:ReleaseAddress",
-							"ec2:ReplaceIamInstanceProfileAssociation",
-							"ec2:RequestSpotInstances",
-							"ec2:RevokeSecurityGroupEgress",
-							"ec2:RevokeSecurityGroupIngress",
-							"ec2:RunInstances",
-							"ec2:TerminateInstances",
-						},
-						Resources: "*",
+	type AwsCrossAccountPolicy struct {
+		PolicyType      string   `json:"policy_type,omitempty" tf:"default:managed"`
+		PassRole        []string `json:"pass_roles,omitempty"`
+		JSON            string   `json:"json,omitempty" tf:"computed"`
```

Comment:
Is `omitempty` needed here? This should not be user-specified. This does cause a schema diff:
```
@ ["provider_schemas","registry.terraform.io/databricks/databricks","data_source_schemas","databricks_aws_crossaccount_policy","block","attributes","json","optional"]
+ true
```

Reply:
fixed